src/components/ha-alert.ts

```diff
@@ -98,7 +98,7 @@ class HaAlert extends LitElement {
       border-radius: 4px;
     }
     .icon {
-      z-index: 1;
+      z-index: 0;
```

Comment:
Why was this z-index set in the first place? And if we are going to set it to 0, can't we just remove the rule?